src/twisted/web/test/test_http.py

```diff
@@ -2569,18 +2568,6 @@ def process(self):
 
 
 class QueryArgumentsTests(unittest.TestCase):
-    # FIXME: https://twistedmatrix.com/trac/ticket/10096
```

Comment:
this test was moved to the ticket https://twistedmatrix.com/trac/ticket/10096#comment:7

as without 3.6 we no longer run it.